src/server/db_slice.cc

```diff
@@ -235,8 +177,7 @@ unsigned PrimeEvictionPolicy::Evict(const PrimeTable::HotspotBuckets& eb, PrimeT
       return 0;
     }
 
-    DbTable* table = db_slice_->GetDBTable(cntx_.db_index);
```

Comment:
Evict here does not records the journal change of item deletion. can you please add it